package.json

```diff
@@ -16,7 +16,6 @@
     "apollo-client": "^2.6.10",
     "axios": "^0.21.1",
     "bignumber.js": "^9.0.1",
-    "caniuse-lite": "^1.0.30001245",
```

Comment:
`npx browserslist --update-db` removed this after running. In the logs I see:

```
Cleaning package.json dependencies from caniuse-lite
$ yarn remove -W caniuse-lite
```

According to https://github.com/browserslist/browserslist#browsers-data-updating this seems like an instance of `caniuse-lite` deduplication